### ISSUE TITLE
Full list of Supported Devices and OS-es

### ISSUE THREAD

**Issue by foxnoodles** [checkra1n/BugTracker#633]
Can you guys put a full list of Supported Devices and OS-s on the website? 
It's getting kinda confusing, people speculating about supported iOS versions as well as devices, it's all over the place. For example, iPod touch support is very unclear. 5th gen to 7th gen?

**Comment by boierito** [comment]
It is easy to understand, only you need to know first if you iDevice has a CPU < A12, second an iOS > 12.3 and final a 64bit CPU.

**Comment by foxnoodles** [comment]
Yeah, I understand that I was coming from an average user's point of view. A simple list is much easier and friendlier to grasp. For everyone including the press.
